docs/proposal-config-changes.md

```diff
@@ -88,16 +78,11 @@ name. See
 See
 [discussion](https://github.com/cylc/cylc-flow/pull/3348#discussion_r333229597).
 
-### `[cylc]task event mail interval` -> `[scheduler][mail]task event interval`
+### `[cylc]task event mail interval` -> `[scheduler][mail]task event batch interval`
```

Comment:
This conforms with my meeting notes, but shouldn't it be `task event mail batch interval`, for complete clarity?  (We don't do any other kind of event batching do we?).

Reply:
It's already in the `[mail]` section- is that not enough?

Reply:
Ah, sorry, missed that. Duh!